### PR TITLE
Add model training logs

### PR DESCRIPTION
**Pull request by HabKaffee**

### CODE REVIEW

File: logs/first_train/log.log [hunk 1]
```diff
@@ -0,0 +1 @@
+zsh: command not found: python
```
Comment (alexgiving) [inline logs/first_train/log.log:1]: rm

**Review by alexgiving — CHANGES_REQUESTED** [review]
Please. delete archives and keep only necessary log files with appropriate name ex: `logs/20th_train/half_classbert_100_32batch_1.log`

**Comment by alexgiving** [comment]
@HabKaffee It is better to make a special `report` folder with `logs` inside

File: logs/first_train/log.log (hunk 1 above)
Comment (HabKaffee) [reply logs/first_train/log.log:1]: Deleted

File: logs/20th_train/night_train.zip
[the remark this answers is not in the document]
Comment (HabKaffee) [reply logs/20th_train/night_train.zip:1]: deleted

File: report/logs/first_train/50_epochs_halfs.log
Comment (alexgiving) [inline report/logs/first_train/50_epochs_halfs.log:1]: please rename it or delete

File: report/logs/first_train/train_10_epochs_fp32.log
Comment (alexgiving) [inline report/logs/first_train/train_10_epochs_fp32.log:1]: same

File: report/logs/first_train/50_epochs_halfs.log
Comment (HabKaffee) [reply report/logs/first_train/50_epochs_halfs.log:1]: done

**Review by alexgiving — APPROVED** [review]

**Pull request merged**